models/reservation.js

```diff
@@ -152,6 +98,21 @@ exports.finish = function(reservationId) {
     });
 };
 
+exports.review = function(reservationId, rating, comment) {
+  var reservation;
+  return exports.updateStatus(reservationId, exports.ReservationStatusEnum.reviewed)
+    .then(function(res) {
+      reservation = res;
+      return Promise.all([
+        Spot.review(reservation.attributes.spotId, reservationId, rating, comment),
+        User.removeReservationFromDriverActive(reservation)
+      ]);
+    })
+    .then(function() {
+      return reservation;
```

Comment:
Should this be:
```
return exports.get(reservationId); 
``` 
instead so that you return the updated one? I guess it probably doesn't matter too much since we're essentially done with the reservation at this point, but just for the sake of consistency it may be good.